keymanapp/keyman
# fix(mac): display package info after keyboard installation

When a keyboard is downloaded and installed, the package information window should be displayed immediately after dismissing the download dialog:

![image](https://github.com/user-attachments/assets/ddaaad20-a3f7-4377-8e3a-5a4650cb6c54)

Fixes #9308

# User Testing

## TEST_PACKAGE_INFO_DISPLAYED

1. Open Keyman Configuration dialog. 
1. Click Download Keyboard button. 
1. Install any keyboard. 
1. Confirm that the Package Information dialog is displayed after clicking the Done button. 

## TEST_DELETE_KEYBOARD

1. Open Keyman Configuration dialog. 
1. For an installed keyboard, click on the delete icon  
1. Confirm that a confirmation dialog is correctly displayed
1. Click the Delete button on the dialog 
1. Confirm that keyboard has been deleted and is no longer listed in the configuration window or in the Keyman menu 

## TEST_BOGUS_KEYBOARD_INSTALL

1. In the Finder, rename a small text file to bogus.kmp
1. Double-click the file 
1. Confirm that an installation confirmation dialog is correctly displayed
1. Click the Install button
1. Confirm that an error alert is displayed, indicating that the .kmp file could not be read
